waiter/test/waiter/token_test.clj

```diff
@@ -1347,7 +1347,7 @@
       (testing "post:new-service-description:invalid-authentication"
         (let [kv-store (kv/->LocalKeyValueStore (atom {}))
               service-description (walk/stringify-keys
-                                    {:cmd "tc1" :cpus 1 :mem 200 :version "a1b2c3" :run-as-user "tu1" :authentication "unsupported"
+                                    {:cmd "tc1" :cpus 1 :mem 200 :version "a1b2c3" :run-as-user "tu1" :authentication ""
```

Comment:
`unsupported` is better than an empty string as input here.

Reply:
this test is for the empty string

Reply:
Is there a separate test for `unsupported` which the previous test was testing?

Reply:
ah, i see.

the previous test is no longer valid. "unsupported" returns 200  there's no longer a restriction on just 2 valid values

Reply:
It shouldn't return 200, the service validation change from core should fail it.

Reply:
this unit test only calls waiter.token/handle-token-request

Reply:
We should have some sort of test for the `unsupported` case, either unit or integration.

Reply:
well, unit test is fine. it will be successful

Reply:
it's acceptable now in this test